test/utils/infer_default_campaign.spec.js

```diff
@@ -0,0 +1,61 @@
+import '../testHelper';
+import { inferDefaultCampaign } from '../../app/assets/javascripts/components/course/utils/inferDefaultCampaign';
+
+const campaigns = [
+  { id: 1, title: 'Spring 2022', slug: 'spring_2022' },
+  { id: 2, title: 'Summer 2022', slug: 'summer_2022' },
+  { id: 3, title: 'Fall 2022', slug: 'fall_2022' }
+];
+let start_date;
+
+describe('No matching campaign', () => {
+  test('returns null', () => {
+    start_date = moment([2019, 2, 5]);
+    const result = inferDefaultCampaign(campaigns, start_date);
+    expect(result).toBe(null);
+  });
+});
+
+describe('Matching campaigns available', () => {
```

Comment:
This is a nice set of tests.

Reply:
Okay. What about the feature tests? Are they fine?

Reply:
Yes, those look good as well.